lib/oeqa/selftest/cases/updater_qemux86_64.py

```diff
@@ -358,7 +360,7 @@ def configure(self):
             self._test_ctx.append_config('SOTA_CLIENT_PROV = " aktualizr-shared-prov "')
 
         def is_ecu_registered(self, ecu_id):
-            max_number_of_tries = 40
+            max_number_of_tries = 120
```

Comment:
I think, I can replace the current implementation that iterates over check & wait with `aktualizr-info --wait-until-provisioned`

Reply:
Sounds like a good plan, but obviously not a high priority. :)